demo.py

```diff
@@ -23,24 +23,29 @@ def main():
   # Define the supply chain
   #############################################################################
   """
+  # Create keys
+  print "Generate keypair for Alice..."
```

Comment:
You might need to include `from __future__ import print_function` at the top of the module to
recognize the print() function.

>
Note
This function is not normally available as a built-in since the name print is recognized as the print statement. To disable the statement and use the print() function, use this future statement at the top of your module:
from __future__ import print_function

https://docs.python.org/2/library/functions.html#print